skypy/galaxy/spectrum.py

```diff
@@ -205,37 +205,37 @@ def kcorrect_spectra(redshift, stellar_mass, coefficients):
 
 
 def mag_ab(spectrum, bandpass, redshift=None):
-    r'''Compute absolute AB magnitude from spectrum and bandpass.
+    r'''Compute absolute AB magnitudes from spectra and bandpasses.
 
-    This function takes an *emission* spectrum and an observation bandpass and
-    computes the AB magnitude for a source at 10pc (i.e. absolute magnitude).
-    The emission spectrum can optionally be redshifted. The definition of the
+    This function takes *emission* spectra and observation bandpasses and
+    computes the AB magnitudes for sources at 10pc (i.e. absolute magnitudes).
+    The emission spectra can optionally be redshifted. The definition of the
     bandpass AB magnitude is taken from [1]_.
 
-    Both the spectrum and the bandpass must be given as functions of wavelength
-    in Angstrom. The spectrum must be given as flux in erg/s/cm2/A.
+    Both the spectra and the bandpasses must be given as functions of
+    wavelength. The spectra must be given as fluxes with units equivalent to
+    erg/s/cm2/A while the bandpasses should be dimensionless.
```

Comment:
This is also no longer true because we use the specutils class which is unitful, and should have been changed earlier.

```suggestion
    The bandpasses should have dimensionless `flux` units.
```